### ISSUE TITLE
Training on coco dataset error

### ISSUE THREAD

**Issue by bluetooth12** [fizyr/keras-retinanet#895]
I get the following error messages when I try to train keras-retinanet on the coco dataset. Anybody have any ideas?
![image](https://user-images.githubusercontent.com/8892431/52433418-e2954980-2ac1-11e9-8a98-1fcc640f8af3.png)

**Comment by martinzlocha** [comment]
I haven't ever trained on the COCO dataset but from the error it seems that the images aren't in the location where they should be.

**Comment by bluetooth12** [comment]
Where are the images supposed to be? I have them stored in coco/images/test2017, coco/images/train2017 and coco/images/val2017, and coco/annotations. When I reduce the epoch and step size from 50 and 1000 to 5 and 10 respectively, the code trains without an error. The problem is that the accuracy decreases and the loss increases. Any ideas why? 
![image](https://user-images.githubusercontent.com/8892431/52588954-8a788300-2df2-11e9-98e0-b4851e35777e.png)

**Comment by bluetooth12** [comment]
When I convert the .h5 file from the last iteration under snapshots to an inference file and then test it, I get no detections/bounding boxes.
![image](https://user-images.githubusercontent.com/8892431/52593245-55256280-2dfd-11e9-99c4-bdbf59151214.png)

**Comment by bluetooth12** [comment]
I increased the step size to 100 and the graphs made more sense. But the no detections are made. 
![image](https://user-images.githubusercontent.com/8892431/52600637-20bba180-2e11-11e9-83ab-941ea44840a9.png)

**Comment by martinzlocha** [comment]
Try using debug.py to see if the bounding boxes look ok there.

**Comment by bluetooth12** [comment]
"python debug.py --annotations coco coco" gives me green boxes that are correctly labelled.

**Comment by hgaiser** [comment]
It seems like at least one of your images is not where your annotations says they should be. Strange that the error doesn't mention the path of the image that it can't find. You can try adding a `print(path)` before [this](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/utils/image.py#L31) line, run it again and see which path is printed before it errors (and then check if that image indeed doesn't exist).

For now though, I'll consider this a non-issue since it is likely in your dataset configuration.

**Comment by bluetooth12** [comment]
I tried printing out the path name and I found the image that it stopped at in its correct position. 
![image](https://user-images.githubusercontent.com/8892431/53188453-39b71600-35ba-11e9-89fd-b68fdfcaee1b.png)
My files are under 
keras-retinanet/coco/images/train2017
keras-retinanet/coco/images/test2017
keras-retinanet/coco/images/val2017
keras-retinanet/coco/annotations/captions_train2017
keras-retinanet/coco/annotations/captions_val2017
keras-retinanet/coco/annotations/instances_train2017
keras-retinanet/coco/annotations/instances_val2017
keras-retinanet/coco/annotations/person_keypoints_train2017
keras-retinanet/coco/annotations/person_keypoints_val2017
Does anybody see any mistakes?

**Comment by bluetooth12** [comment]
I am running python keras_retinanet/bin/train.py --weights resnet50_coco_best_v2.1.0.h5 coco coco

keras-retinanet/coco/images/train2017/images...
keras-retinanet/coco/images/test2017/images...
keras-retinanet/coco/images/val2017/images...
keras-retinanet/coco/annotations/captions_train2017.json
keras-retinanet/coco/annotations/captions_val2017.json
keras-retinanet/coco/annotations/instances_train2017.json
keras-retinanet/coco/annotations/instances_val2017.json
keras-retinanet/coco/annotations/person_keypoints_train2017.json
keras-retinanet/coco/annotations/person_keypoints_val2017.json

I have tried without weights and got the same result.
